modules/profile.nix

```diff
@@ -15,9 +15,18 @@
     "fsck.repair=yes"
     "net.ifnames=0"
     "elevator=noop"
-    "ethaddr=\${ethaddr}"
     "enable_wol=1"
     "usb-xhci.tablesize=2"
+    "maxcpus=6"
+    # "max_freq_a53=1896" # n2
+    # "max_freq_a73=1800" # n2
+    "max_freq_a53=1908" # n2-plus
```

Comment:
Could you switch on the freq a flag like `n2plus`? I still have the older revisions.

Reply:
I'm not even sure if this makes a difference :-) I've been testing this on an old (non-plus). I'm mildly confused about the actual meaning of this flag.